perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -192,6 +207,10 @@
 }
 AWS_S3_ENDPOINT_SUFFIX = '.amazonaws.com'
 
+MULTISTREAM_FIXED_DELAY = 5.0
```

Comment:
Please add more comment to clarify why do we need these two delays, and what each one is for. My understanding is:

Multistream_fixed_delay is to accomondate multiple VM start time differences. it covers the most likely time it takes to launch remote commands to N VMs.

Multistream_delay_per_stream is to acommondate multiple stream start time differences on a single VM.

If my understanding is correct, two follow-up items:

1) please consider rename these variables to reflect this.
2) the 2 second per stream overhead is too high. This number is to accomondate  per thread start overhead so the last thread started can start write at the same time as the first thread that's started. I don't think it's going to take 2 seconds to start one thread. If I have to estimate, it will be far less than 100ms or so.